tests_with_redis.sh

```diff
@@ -29,11 +29,12 @@ then
 elif [ "$1" == "integrationtest" ]
 then
   pytest -m 'not unittest and not noauth'
-  if [ $? -eq 0]
+  INTEGRATIONTEST_RETURN=$?
+  if [ ${INTEGRATIONTEST_RETURN} -eq 0]
   then
     run_tests_noauth
   else
-    echo "Skipping tests without authentication"
+    echo "Skipping tests without authentication scince other tests failed with ${INTEGRATIONTEST_RETURN}"
```

Comment:
```suggestion
    echo "Skipping tests without authentication since other tests failed with ${INTEGRATIONTEST_RETURN}"
```

This change is not clear to me: "... failed with 1" has been added?

Reply:
It is only for debugging why the tests are not running. The exit code can be something between 0-5 (https://docs.pytest.org/en/7.1.x/reference/exit-codes.html)

Reply:
I see - so only the typo fix remains :)